controllers/admin/AdminStoresController.php

```diff
@@ -566,6 +566,17 @@ public function beforeUpdateOptions()
         }
     }
 
+    public function updateOptionPsShopName($value)
```

Comment:
```suggestion
    /**
     * @var string $name
     */
    public function updateOptionPsShopName(string $name): void
```

Could you improve the docstring to explained what this method do? :)